codegen/Cargo.toml

```diff
@@ -7,18 +7,19 @@ edition = "2018"
 [lib]
 
 [dependencies]
-autorust_openapi = { git = "https://github.com/ctaggart/autorust_openapi" }
+autorust_openapi = { rev = "181baf2b89d32c013258e40b59d3e00bb501c9c4", git = "https://github.com/ctaggart/autorust_openapi" }
 # autorust_openapi = { path = "../autorust_openapi" }
 quote = "*"
 proc-macro2 = { version = "*", default-features = false }
 serde_json = "*"
 serde_yaml = "*"
 heck = "*"
 regex = "*"
-indexmap = {version = "*", features = ["serde-1"]}
+indexmap = { version = "*", features = ["serde-1"] }
 path_abs = "*"
 comrak = "0.8"
 serde = "1.0"
-snafu = "0.6"
+snafu = { version = "0.6", features = ["backtraces"] }
```

Comment:
I didn't find backtraces useful after added the other errors. the backtrace library was definitely further along than the nightly std support.